minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -246,7 +244,8 @@ public void start(TachyonConf conf) throws IOException {
     // Disable hdfs client caching to avoid file system close() affecting other clients
     System.setProperty("fs.hdfs.impl.disable.cache", "true");
 
-    startMaster(conf);
+    MasterContext.getConf().merge(conf);
```

Comment:
It's intended since other PRs I made depend on this class, so I didn't modify its public methods, I'll leave todo to remove the unnecessary methods.